src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient/Models/Arkivmelding.cs

```diff
@@ -0,0 +1,412 @@
+        /// </summary>
+        [XmlElement(ElementName = "tittel")]
+        public string Tittel { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the OpprettetDato
+        /// </summary>
+        [XmlElement(ElementName = "opprettetDato")]
+        public DateTime OpprettetDato { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the OpprettetAv
+        /// </summary>
+        [XmlElement(ElementName = "opprettetAv")]
+        public string OpprettetAv { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the TilknyttetRegistreringSom
+        /// </summary>
+        [XmlElement(ElementName = "tilknyttetRegistreringSom")]
+        public string TilknyttetRegistreringSom { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Dokumentnummer
+        /// </summary>
+        [XmlElement(ElementName = "dokumentnummer")]
+        public int Dokumentnummer { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the TilknyttetDato
+        /// </summary>
+        [XmlElement(ElementName = "tilknyttetDato")]
+        public DateTime TilknyttetDato { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the TilknyttetAv
+        /// </summary>
+        [XmlElement(ElementName = "tilknyttetAv")]
+        public string TilknyttetAv { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the Dokumentobjekt
+        /// </summary>
+        [XmlElement(ElementName = "dokumentobjekt")]
+        public Dokumentobjekt Dokumentobjekt { get; set; }
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Korrespondansepart"/> class.
+    /// </summary>
+    [XmlRoot(ElementName = "korrespondansepart")]
+    public class Korrespondansepart
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Korrespondansepart"/> class.
+        /// </summary>
+        /// <param name="korrespondanseparttype">Korrespondanseparttype</param>
+        /// <param name="korrespondansepartNavn">KorrespondansepartNavn</param>
+        public Korrespondansepart(string korrespondanseparttype, string korrespondansepartNavn)
+        {
+            Korrespondanseparttype = korrespondanseparttype;
+            KorrespondansepartNavn = korrespondansepartNavn;
+        }
+
+        /// <summary>
+        ///  Gets or sets the Korrespondanseparttype
+        /// </summary>
+        [XmlElement(ElementName = "korrespondanseparttype")]
+        public string Korrespondanseparttype { get; set; }
+
+        /// <summary>
+        ///  Gets or sets the KorrespondansepartNavn
+        /// </summary>
+        [XmlElement(ElementName = "korrespondansepartNavn")]
+        public string KorrespondansepartNavn { get; set; }
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Basisregistrering"/> class.
+    /// </summary>
+    [XmlRoot(ElementName = "basisregistrering")]
+    public class Basisregistrering
```

Comment:
Are all these clases standard classes that everyone requires? I thought we would let the app developer do most of the heavy lifting realted to the archive message ?